### ISSUE TITLE
No Twitch games or platform

### ISSUE THREAD

**Issue by CatCubed** [nyash-qq/galaxy-plugin-twitch#23]
When I downloaded and installed this I was able to connect the integration just like any other but none of my Twitch games are showing up. Also can't see Twitch as a platform in sorting (in the automatically generated Twitch category I can see that it's from a platform but there is no Twitch in the platforms list).

I was using the older version before this (the one that used Amazon and that one worked for me) so maybe there's some issue with that even though I disconnected it and removed it before adding the Twitch one.

**Comment by Rall3n** [comment]
@Lada1208 No Plattform usually means that the plugin failed initialization/authentication.

Do you have the Twitch App installed and have you logged into the Twitch App at least once?

Could you provide the latest plugin log file? It should be located in `C:\ProgramData\GOG.com\Galaxy\logs` under the name `plugin-twitch-8b831aed-dd5f-c0c5-c843-41f9751f67a2.log`?

**Comment by CatCubed** [comment]
Yes, I have the Twitch App installed and logged in. Here's the log file
[plugin-twitch-8b831aed-dd5f-c0c5-c843-41f9751f67a2.log](https://github.com/nyash-qq/galaxy-plugin-twitch/files/3819245/plugin-twitch-8b831aed-dd5f-c0c5-c843-41f9751f67a2.log)

**Comment by Rall3n** [comment]
@nyash-qq It seems like the Twitch App does not create the `GameInstallInfo.sqlite` on a fresh install. It will only be created if a game has been successfully installed for the first time.

**Comment by CatCubed** [comment]
Just installed a game on the Twitch launcher and you're right! Now I can see all my Twitch games and I can play the ones I have installed but I can't seem to be able to install them. When I hover over the greyed out install button it says "Not compatible"

**Comment by Rall3n** [comment]
> Just installed a game on the Twitch launcher and you're right! Now I can see all my Twitch games and I can play the ones I have installed but I can't seem to be able to install them. When I hover over the greyed out install button it says "Not compatible"

@Lada1208 #4
